### DISCUSSION TITLE
Is there a curve control similar to this one

### DISCUSSION THREAD

**Discussion by wosiyourfather** [AvaloniaUI/Avalonia#12537] [Q&A] [answered]
![nwe](https://github.com/AvaloniaUI/Avalonia/assets/53590491/e0b9bcfb-5367-4bd2-bee8-433e33d25173)

**Comment by timunie** [comment, marked as answer]
check Awsome.Avalonia repo if anything similar is there. https://github.com/AvaloniaCommunity/awesome-avalonia

**Comment by bsampica** [comment]
Check out LiveCharts2 - it'll do exactly what you want I think.
https://lvcharts.com/docs/WPF/2.0.0-beta.330/gallery
